apps/meteor/app/apps/server/bridges/livechat.ts

```diff
@@ -180,7 +180,7 @@ export class AppLivechatBridge extends LivechatBridge {
 		return Promise.all(result.map((room) => this.orch.getConverters()?.get('rooms').convertRoom(room) as Promise<ILivechatRoom>));
 	}
 
-	protected async createVisitor(visitor: IVisitor, appId: string): Promise<string> {
+	protected async createVisitor(visitor: IVisitor, appId: string): Promise<IVisitor | undefined> {
```

Comment:
I think we should create a bridge to this, otherwise this would be a breakchange.

its fine to add a new method and make the app only accept newer versions.

but the opposite can be tricky, imagine all the apps that use this bridge will no longer be compatilble